CONTRIBUTING.md

```diff
@@ -1,16 +1,57 @@
 # Contributing
+To make contributions to this charm, you'll need a working Juju development setup.
 
-To make contributions to this charm, you'll need a working [Juju development setup](https://juju.is/docs/sdk/dev-setup).
+## Prerequisites
+### Charmcraft installation
+Charmcraft is a command line tool used alongside the Charmed Operator Framework to create, build, and publish charms.
 
-You can use the environments created by `tox` for development:
+Charmcraft depends on LXD to build the charms in a container matching the target base(s).
```

Comment:
Again, no need for explanation, think of a recipe. Add X, Add Y, Mix X and Y, ...

Reply:
This may apply elsewhere

Reply:
This comment still applies